test/tests/val-2d/tests

```diff
@@ -0,0 +1,45 @@
+[Tests]
+  design = 'ADDirichletBC.md'
+  issues = '#12 #203'
+  validation = 'val-2d.md'
+  [val-2d_csvdiff]
+    type = CSVDiff
+    input = 'val-2d.i'
+    cli_args = 'Mesh/active=cartesian_mesh_coarse Executioner/TimeStepper/dt=1 Outputs/file_base=val-2d_limited_out'
+    csvdiff = val-2d_limited_out.csv
+    recover = false
+    requirement = 'The system shall be able to model thermal desorption spectroscopy on Tungsten.'
+  []
+  [val-2d_exodiff]
+    type = Exodiff
+    input = 'val-2d.i'
+    cli_args = 'Mesh/active=cartesian_mesh_coarse Executioner/TimeStepper/dt=1 Outputs/file_base=val-2d_limited_out'
+    exodiff = val-2d_limited_out.e
+    recover = false
```

Comment:
Add issues number as reference. I'm surprised you need this here because you do not use any object that I would expect would need it. Are you sure it fails recover, and are you sure it's not just an easy fix?